src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java

```diff
@@ -92,6 +92,10 @@ public AccountAttributes build() {
             return accountAttributes;
         }
 
+        public AccountAttributes buildWithoutSanitizing() {
```

Comment:
I don't think it is a good practice to introduce this method in production code just for testing.